Create towercrush-backend.md (#203)
Gamify-IT/docs: user-manuals/minigames/towercrush.md

# Towercrush

Towercrush is a minigame which is part of the Gamify-IT platform.

## About the Game

> The goal of the game is to provide a simple game for students to practice their knowledge with single choice questions while competing against fellow students.

Author: We kept the same style we used in the chickenshock user manual, if I should change both let me know, else I will keep this style consistent

> The goal of the game is to provide a simple game for students to practice their knowledge with single choice questions while competing against fellow students.

Reviewer: Fine for now, will improve it later on.

> ![main menu screenshot](assets/towercrush-game.webp)

Reviewer: Screenshot missing

Author: Will be added once we have improved the design of the game

When entering the game, the user is presented with a lobby search screen.
Once the user joined a lobby they can choose one of two teams to join and can ready up.
If every user in the lobby has chosen a team and readied up, the players can start the game by clicking the Start button.
The users can only go to the next question once every member of the team has voted for their preferred answer.
Teams get rewarded with points for every question they answered correctly, the aim of the game is to keep the tower of your team alive by getting as many points as possible.

Once a team has answered every question, or one of the team's towers reaches zero the game ends.

The winning team is the one whose tower is the last one standing, or whichever team has answered more questions correctly.

> ![game screenshot](assets/towercrush-end-screen.webp)

Reviewer: Screenshot missing

Reviewer: Additionally:
```suggestion
![game end screenshot](assets/towercrush-end-screen.webp)
```

> When all questions are answered, the winning team is presented.

Reviewer: ```suggestion
When all questions are answered, you will be notified which team won.
```
